docs/Resources-Appendices/ascii-collating-sequence.md

```diff
@@ -4,5 +4,50 @@ sidebar_position: 1
+| 00001111 | 0F   | 15   | SI     |     | 00111010 | 3A   | 58   | :      |     | 01100101 | 65   | 101  | e      |
+| 00010000 | 10   | 16   | DLE    |     | 00111011 | 3B   | 59   | ;      |     | 01100110 | 66   | 102  | f      |
+| 00010001 | 11   | 17   | DC1    |     | 00111100 | 3C   | 60   | <      |     | 01100111 | 67   | 103  | g      |
+| 00010010 | 12   | 18   | DC2    |     | 00111101 | 3D   | 61   | =      |     | 01101000 | 68   | 104  | h      |
+| 00010011 | 13   | 19   | DC3    |     | 00111110 | 3E   | 62   | >      |     | 01101001 | 69   | 105  | i      |
+| 00010100 | 14   | 20   | DC4    |     | 00111111 | 3F   | 63   | ?      |     | 01101010 | 6A   | 106  | j      |
+| 00010101 | 15   | 21   | NAK    |     | 01000000 | 40   | 64   | @      |     | 01101011 | 6B   | 107  | k      |
+| 00010110 | 16   | 22   | SYN    |     | 01000001 | 41   | 65   | A      |     | 01101100 | 6C   | 108  | l      |
+| 00010111 | 17   | 23   | ETB    |     | 01000010 | 42   | 66   | B      |     | 01101101 | 6D   | 109  | m      |
+| 00011000 | 18   | 24   | CAN    |     | 01000011 | 43   | 67   | C      |     | 01101110 | 6E   | 110  | n      |
+| 00011001 | 19   | 25   | EM     |     | 01000100 | 44   | 68   | D      |     | 01101111 | 6F   | 111  | o      |
+| 00011010 | 1A   | 26   | SUB    |     | 01000101 | 45   | 69   | E      |     | 01110000 | 70   | 112  | p      |
+| 00011011 | 1B   | 27   | ESC    |     | 01000110 | 46   | 70   | F      |     | 01110001 | 71   | 113  | q      |
+| 00011100 | 1C   | 28   | FS     |     | 01000111 | 47   | 71   | G      |     | 01110010 | 72   | 114  | r      |
+| 00011101 | 1D   | 29   | GS     |     | 01001000 | 48   | 72   | H      |     | 01110011 | 73   | 115  | s      |
+| 00011110 | 1E   | 30   | RS     |     | 01001001 | 49   | 73   | I      |     | 01110100 | 74   | 116  | t      |
+| 00011111 | 1F   | 31   | US     |     | 01001010 | 4A   | 74   | J      |     | 01110101 | 75   | 117  | u      |
+| 00100000 | 20   | 32   | SP     |     | 01001011 | 4B   | 75   | K      |     | 01110110 | 76   | 118  | v      |
+| 00100001 | 21   | 33   | !      |     | 01001100 | 4C   | 76   | L      |     | 01110111 | 77   | 119  | w      |
+| 00100010 | 22   | 34   | "      |     | 01001101 | 4D   | 77   | M      |     | 01111000 | 78   | 120  | x      |
+| 00100011 | 23   | 35   | #      |     | 01001110 | 4E   | 78   | N      |     | 01111001 | 79   | 121  | y      |
+| 00100100 | 24   | 36   | $      |     | 01001111 | 4F   | 79   | O      |     | 01111010 | 7A   | 122  | z      |
+| 00100101 | 25   | 37   | %      |     | 01010000 | 50   | 80   | P      |     | 01111011 | 7B   | 123  | {      |
+| 00100110 | 26   | 38   | &      |     | 01010001 | 51   | 81   | Q      |     | 01111100 | 7C   | 124  |<img width="11" alt="Screen Shot 2021-11-15 at 5 51 05 PM" src="https://user-images.githubusercontent.com/65831678/141865226-06c0f43b-a52f-4f7a-8b74-97398a427163.png">      |
```

Comment:
If you're going to use an image, it needs to live in the repo (where we control it) vs in an external site.

Reply:
I don't have any idea how to add | into the table. Any advice?

Reply:
If you think it supposes to be an image, too, I will redo it now.

Reply:
Fixed, with \|